provisioner/clusterpy.go

```diff
@@ -365,29 +369,35 @@ func (p *clusterpyProvisioner) Provision(ctx context.Context, logger *log.Entry,
 	return p.apply(ctx, logger, cluster, deletions, manifests)
 }
 
-func createOrUpdateClusterStack(ctx context.Context, config channel.Config, cluster *api.Cluster, values map[string]interface{}, adapter *awsAdapter, bucketName string) error {
+func createOrUpdateClusterStack(ctx context.Context, config channel.Config, cluster *api.Cluster, values map[string]interface{}, adapter *awsAdapter, bucketName string) ([]*cloudformation.Output, error) {
 	template, err := config.StackManifest(clusterStackFileName)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	rendered, err := renderSingleTemplate(template, cluster, nil, values, adapter)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	err = adapter.applyClusterStack(cluster.LocalID, rendered, cluster, bucketName)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	ctx, cancel := context.WithTimeout(ctx, maxWaitTimeout)
 	defer cancel()
 	err = adapter.waitForStack(ctx, waitTime, cluster.LocalID)
 	if err != nil {
-		return err
+		return nil, err
 	}
-	return nil
+
+	clusterStack, err := adapter.getStackByName(cluster.LocalID)
+	if err != nil {
+		return nil, err
+	}
+
+	return clusterStack.Outputs, nil
```

Comment:
Maybe return a map of string instead over here.

Reply:
Nice idea, changed it to a map.